app/models/current_logging_attributes.rb

```diff
@@ -15,5 +15,7 @@ class CurrentLoggingAttributes < ActiveSupport::CurrentAttributes
             :answer_type,
             :auth0_session_id,
             :validation_errors,
-            :params
+            :params,
+            :queries_count,
+            :cached_queries_count
```

Comment:
If we're adding the attributes to the `attributes` hash with merge we don't need to define them here